indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -175,3 +206,9 @@ export default function EditableList() {
     </>
   );
 }
+
+EditableListDisplay.propTypes = {
+  editableList: PropTypes.array.isRequired,
+  editableType: PropTypes.string.isRequired,
+  eventId: PropTypes.any.isRequired,
```

Comment:
why `any`? this should always be a `number`..

Reply:
Ohh, I forgot to change this.